File: Data Structures Algorithns/JavaScript/sorting.js
export function selection_sort(arr) {
  for (let i = 0; i < arr.length; i++) {
    let min = arr[i];
    let pos = i;
    for (let j = i + 1; j < arr.length; j++) {
      if (arr[j] < min) {
        min = arr[j];
        pos = j;
      }
    }
    if (min !== arr[i]) {
      temp = arr[pos];
      arr[pos] = arr[i];
      arr[i] = temp;
    }
  }
  return arr;
}

File: Data Structures Algorithns/Python/sorting.py
def selection_sort(arr):
    for i in range(len(arr)):
        min, pos = arr[i], i
        for j in range(i+1,len(arr)):
            if arr[j] < min:
                min, pos = arr[j], j
        if not arr[i] == min:
            arr[i], arr[pos] = arr[pos], arr[i]
    return arr